Testnet command error: MustUnmarshalJSON panic

When use `evmosd testnet init-files --output-dir ./mytestnet --chain-id evmos_9000-1 --keyring-backend file --v 1` 
it will be panic as follows : 

panic: EOF
github.com/cosmos/cosmos-sdk/codec.(*ProtoCodec).MustUnmarshalJSON(0x140005b05d0, {0x0, 0x0, 0x0}, {0x106b42030, 0x14001077090})
	github.com/cosmos/cosmos-sdk@v0.45.1/codec/proto_codec.go:176 +0x80
main.initGenFiles({{0x0, 0x0, 0x0}, {0x106baf218, 0x1400104f680}, {0x16b28b4be, 0xc}, {0x106b7b428, 0x140005b05d0}, {0x106ba3ab8, ...}, ...}, ...)
	github.com/tharsis/evmos/v2/cmd/evmosd/testnet.go:415 +0x6a8
main.initTestnetFiles({{0x0, 0x0, 0x0}, {0x106baf218, 0x1400104f680}, {0x16b28b4be, 0xc}, {0x106b7b428, 0x140005b05d0}, {0x106ba3ab8, ...}, ...}, ...)

According to error stack information , found that module mintGenState doesn't seem to be used .  
So should it be remove ?

--- a/cmd/evmosd/testnet.go
+++ b/cmd/evmosd/testnet.go
@@ -34,7 +34,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/genutil"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
-	mintypes "github.com/cosmos/cosmos-sdk/x/mint/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 
 	"github.com/tharsis/ethermint/crypto/hd"
@@ -410,12 +409,6 @@ func initGenFiles(
 
 	govGenState.DepositParams.MinDeposit[0].Denom = coinDenom
 	appGenState[govtypes.ModuleName] = clientCtx.Codec.MustMarshalJSON(&govGenState)
-
-	var mintGenState mintypes.GenesisState
-	clientCtx.Codec.MustUnmarshalJSON(appGenState[mintypes.ModuleName], &mintGenState)
-
-	mintGenState.Params.MintDenom = coinDenom
-	appGenState[mintypes.ModuleName] = clientCtx.Codec.MustMarshalJSON(&mintGenState)
 
 	var crisisGenState crisistypes.GenesisState
 	clientCtx.Codec.MustUnmarshalJSON(appGenState[crisistypes.ModuleName], &crisisGenState)

